src/basics.md

```diff
@@ -412,6 +414,80 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-glob-recursive]: #ex-glob-recursive
+<a name="ex-glob-recursive"></a>
+## Find all png files recursively
+
+[![glob-badge]][glob] [![cat-filesystem-badge]][cat-filesystem]
+
+Recursively find all png files using the [glob] crate.
+The `**` pattern matches the current directory and all subdirectories.
+
+```rust
+# #[macro_use]
+# extern crate error_chain;
+extern crate glob;
+
+use glob::glob;
+#
+# error_chain! {
+#     foreign_links {
+#         Glob(glob::GlobError);
+#         Pattern(glob::PatternError);
+#     }
+# }
+
+fn run() -> Result<()> {
+    for entry in glob("**/*.png")? {
+        println!("{}", entry?.display());
+    }
+
+    Ok(())
+}
+#
+# quick_main!(run);
+```
+
+[ex-glob-with]: #ex-glob-with
+<a name="ex-glob-with"></a>
+## Find all files with given pattern ignoring filename case.
+
+[![glob-badge]][glob] [![cat-filesystem-badge]][cat-filesystem]
+
+Find all image files in the `/media/` directory matching the `img_[0-9]*.png` pattern using the [glob] crate.
```

Comment:
I would suggest dropping the `using glob crate part`. Reader should be already informed about it thanks to badges/code snippet and other links in the description.